[2024-11-19T04:34:22.995+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_reconciliation_example.load_data manual__2024-11-19T04:34:19.823534+00:00 [queued]>
[2024-11-19T04:34:23.005+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_reconciliation_example.load_data manual__2024-11-19T04:34:19.823534+00:00 [queued]>
[2024-11-19T04:34:23.006+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-11-19T04:34:23.020+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): load_data> on 2024-11-19 04:34:19.823534+00:00
[2024-11-19T04:34:23.025+0000] {standard_task_runner.py:60} INFO - Started process 51897 to run task
[2024-11-19T04:34:23.030+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'data_reconciliation_example', 'load_data', 'manual__2024-11-19T04:34:19.823534+00:00', '--job-id', '11667', '--raw', '--subdir', 'DAGS_FOLDER/data_reconciliation_example.py', '--cfg-path', '/tmp/tmp80lf2en3']
[2024-11-19T04:34:23.031+0000] {standard_task_runner.py:88} INFO - Job 11667: Subtask load_data
[2024-11-19T04:34:23.087+0000] {task_command.py:423} INFO - Running <TaskInstance: data_reconciliation_example.load_data manual__2024-11-19T04:34:19.823534+00:00 [running]> on host 3c704ad499e6
[2024-11-19T04:34:23.164+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_reconciliation_example' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-19T04:34:19.823534+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-19T04:34:19.823534+00:00'
[2024-11-19T04:34:23.165+0000] {logging_mixin.py:188} INFO - Data reconciled! Proceeding with loading data to final table...
[2024-11-19T04:34:23.166+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-11-19T04:34:23.175+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=data_reconciliation_example, task_id=load_data, execution_date=20241119T043419, start_date=20241119T043422, end_date=20241119T043423
[2024-11-19T04:34:23.201+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-19T04:34:23.221+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
